hazelcast/src/test/java/com/hazelcast/map/PartitionPredicateTest.java

```diff
@@ -111,6 +128,43 @@ public void entries() {
         }
     }
 
+    @Test
```

Comment:
Can you add the same tests on the `ClientPartitionPredicateTest` as well?

Reply:
@mmedenjak `ClientPartitionPredicateTest ` contains similar tests for `aggregate`, `project` and  `executeOnEntries` already. @sancar fixed the client side in 3.9.2. See https://github.com/hazelcast/hazelcast/pull/11961